org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/handlers/ResolveClasspathsHandler.java

```diff
@@ -125,7 +125,7 @@ private static String computeClassPath(String projectName, String mainClass) thr
 		} else {
 			List<IJavaProject> projects = getJavaProjectFromType(mainClass);
 			if (projects.size() == 0 || projects.size() > 1) {
-				throw new CoreException(new Status(IStatus.ERROR, JavaLanguageServerPlugin.PLUGIN_ID, "project count is zero or more than one."));
+				throw new CoreException(new Status(IStatus.ERROR, JavaLanguageServerPlugin.PLUGIN_ID, String.format("main class %s doesn't exist or isn't unique in the workspace", mainClass)));
```

Comment:
Suggest quoting the class name \"\s\" to isolate user input from hard coded message body.